tinygrad/tensor.py

```diff
@@ -204,7 +219,7 @@ def matmul(self:Tensor, w:Tensor):
   dot = matmul
 
   # (padding_left, padding_right, padding_top, padding_bottom)
-  def pad2d(self, padding:Tuple[int, ...]): return self[:, :, -padding[2]:self.shape[2]+padding[3], -padding[0]:self.shape[3]+padding[1]]
+  def pad2d(self, padding:Tuple[int, ...]): return self.slice(arg = [(0,self.shape[0]), (0,self.shape[1]), (-padding[2],self.shape[2]+padding[3]), (-padding[0],self.shape[3]+padding[1])])  # type: ignore
```

Comment:
Getting close! Why is this type: ignore?

Reply:
mypy doesn't seem to know about the methods that are registered dynamically, so it complains that Tensor has no attribute "slice`. I was looking for a generic way to tell mypy about these methods since they're referenced quite a bit in Tensor, but I haven't been able to find a concise solution yet.